src/OrleansProviders/Streams/Common/PooledCache/PooledQueueCache.cs

```diff
@@ -33,6 +34,8 @@ public class PooledQueueCache<TQueueMessage, TCachedMessage>
         private readonly CachedMessagePool<TQueueMessage, TCachedMessage> pool;
         private readonly ICacheDataAdapter<TQueueMessage, TCachedMessage> cacheDataAdapter;
         private readonly ICacheDataComparer<TCachedMessage> comparer;
+        private readonly Logger logger;
+        private int itemCount;
```

Comment:
I imagine int is enough, but just wanted to confirm with you, maybe there's scenarios when we would want long?
Also, all access to this is without Interlocked, I assume this doesn't need to be thread safe?